src/requirements/requirement-frontend-computation.ts

```diff
@@ -105,20 +105,14 @@ const getTotalCreditsFulfillmentStatistics = (
   }
 
   let minCountFulfilled = 0;
-  let minCountRequired = 120;
-  const courseCodeSet = new Set<string>();
+  const minCountRequired = 120;
   const eligibleCourses =
     college === 'AG'
       ? courses.filter(course => !course.code.startsWith('PE '))
       : courses.filter(courseIsAllEligible);
 
   eligibleCourses.forEach(course => {
     minCountFulfilled += course.credits;
-    if (courseCodeSet.has(course.code)) {
-      minCountRequired += course.credits;
-    } else {
-      courseCodeSet.add(course.code);
-    }
```

Comment:
By removing this code, we do get rid of the issue where the 120 denominator does not increase for repeating courses. However, this means that repeated courses that aren't project teams or research (e.g. having CS 1110 more than once in your plan) would incorrectly get counted towards fulfilling the 120. For example, if I had CS 1110 in my plan twice, my total academic credits fulfillment would be 8/120 instead of 4/120.

@einc, @willespencer should we filter `eligibleCourses` further so that we only increase `minCountFulfilled` if it is a (dupe) project team/research course or if it's the first time we see the course (e.g. counting for the first CS 1110 we see)? Note this could get hairy for non research/project team courses like INFO 4940 where different sections are different courses. The question is how much inaccuracy are we comfortable with? Do we want to overcount/undercount?

Reply:
Yeah I'm leaning toward leaving this as is so that we overcount? I was specifically looking at the [Arts Requirement](https://courses.cornell.edu/content.php?catoid=41&navoid=11570#credit-req) which states specifically 

"Students occasionally need to repeat courses. Some courses, such as independent study, some music and performance courses, and specific topical seminars, in which content is significantly different, do grant credit when the course is taken more than once. For all repeated courses, both grades appear on the transcript and are included in both the term and cumulative GPA.  For repeated courses that do not grant credit more than once, only one instance counts toward degree credits and requirements."

This seems to complicated for us to restrict, and I think we'd be likely to disallow something that does count. Let me know if you think otherwise though!

Reply:
I would be ok with leaving as is and overcounting. In that case, I think we should add a sentence in the description (or a warning?) that says something like: "Repeated courses may not apply to the 120 Academic Credits requirement.". Just so that users are aware of the fact that we don't handle this.